vcpkg-test/vcpkg.json

```diff
@@ -10,5 +10,11 @@
       "host": true
     },
     "tfc-framework"
+  ],
+  "overrides": [
+    {
+      "name": "mp-units",
+      "version": "0.8.0"
+    }
```

Comment:
This is horrible.

mp-units has announced they will not support the 0.8.0 tree
after the release of 2.0 so we should try and update quickly.